src/mixins/trackHelper.js

```diff
@@ -31,6 +31,10 @@ export var getTrackCSS = function(spec) {
     msTransform: 'translateX(' + spec.left + 'px)'
   };
 
+  if (!-[1,]) {
```

Comment:
You can't use window.addEventListener to test IE8 anymore, since the polyfill fixes everything. And we should assume any polyfill could possibly be included in any project. So here we use a native JS feature detection.

Credit: http://stackoverflow.com/a/30617541/1849458